app/permissions/decidim/decidim_awesome/admin/permissions.rb

```diff
@@ -13,8 +13,19 @@ def permissions
 
           toggle_allow(config_enabled?(permission_action.subject)) if permission_action.action == :edit_config
 
+          case permission_action.action
+          when :index
+            admin_accountability_enabled?
+          end
+
           permission_action
         end
+
+        private
+
+        def admin_accountability_enabled?
+          toggle_allow(Decidim::DecidimAwesome.allow_admin_accountability)
```

Comment:
```suggestion
          toggle_allow(Decidim::DecidimAwesome.allow_admin_accountability == true)
```
Let's force to be `true` only, as many confs can be `:disabled` and that counts as false.